src/lewis/adapters/stream.py

```diff
@@ -633,12 +668,18 @@ class StreamInterface(InterfaceBase):
 
     Many hardware devices use a protocol that is based on exchanging text with a client via
     a TCP stream. Sometimes RS232-based devices are also exposed this way via an adapter-box.
-    This adapter makes it easy to mimic such a protocol, in a subclass only three members must
-    be overridden:
+    This adapter makes it easy to mimic such a protocol.
+
+    This class has the following attributes which may be overridden by subclasses:
 
+     - protocol: What this interface is called for purposes of the -p commandline option.
+       Defaults to "stream".
      - in_terminator, out_terminator: These define how lines are terminated when transferred
        to and from the device respectively. They are stripped/added automatically.
-       The default is ``\\r``.
+       Inverse of protocol file InTerminator and OutTerminator. The default is ``\\r``.
+     - readtimeout: How many msec to wait for additional data between packets, once transmission
```

Comment:
But then again, it was mentioned in this comment, should have read more carefully. I really like the sentence from the PR description that says "readtimeout can be considered the terminator if no `in_terminator` has been specified" - that would really make the reason behind this parameter easier to understand.

Related to that, would it be more explicit to require the `in_terminator` set to `None` in that case, or does that break the workings of `asynchat`?